README.md

```diff
@@ -35,25 +35,38 @@ Sugarcoat was created to enable developers to produce rich UI documentation easi
 
 # Features #
 
-1. What does it look like?
+1. You decide where your library lives
 
-   A [Sample Site](https://sapientnitrola.github.io/sugarcoat/) and its accompanying [Pattern Library](https://sapientnitrola.github.io/sugarcoat/documentation/index.html). These pages can be found on your local copy by following a short series of steps: [Example Library](#example-library).
+  Sugarcoat will never impose a folder structure on you.
 
-2. Can you parse css-preprocessor variables?
+2. No extra work
 
-   Yes, we're able to grab the variables from less, SASS, or regular CSS files. Just set the `type` option to `variable` in the appropriate `section` object within the `sections` array. See [`section.type`](#type) for more information.
+  You don't have to create any extra files outside of your files for your pattern library. Just create your config and document your project files as you normally would.
```

Comment:
Love your revision! It seems more concise and describes a single feature that affects your project in multiple ways. Also agree with the "lives in your project seamlessly"